app/src/androidTest/java/com/github/multimatum_team/multimatum/MainActivityTest.kt

```diff
@@ -124,24 +124,24 @@ class MainActivityTest {
         )
     }
 
-    @Test
-    fun buttonOpensQrCodeReader() {
```

Comment:
Do you have a vague idea of what the problem for these tests ?

Reply:
The problem comes from the `grantPermission` and `denyPermission` methods. It may be something like that an incorrect text used to match the widget, but I am not sure about it.